aula/apps/sortides/filters.py

```diff
@@ -61,11 +61,9 @@ def __init__(self, *args, tipus=None, **kwargs):
 
         extra_divs = [
             Div(Field("ambit", css_class="form-control"), css_class="col-lg-4"),
-            Div(Field("tipus", css_class="form-control"), css_class="col-lg-4"),
         ]
         if tipus == "P":
             # remove 'tipus' from filter fields
-            self.filters.pop("tipus")
             self.filters.pop("ciutat")
```

Comment:
Amb pagaments genera error (visible al log de l'apache):
KeyError: "Key 'ciutat' not found in 'SortidaFilterForm'. Choices are: ambit, data_inici_desde, data_inici_fins, empresa_de_transport, estat, mitja_de_transport, professor, subtipus, tipus, titol."
Suposo que és a causa del pop("ciutat").

Reply:
He fet dos filtres, un per Sortides i l'altre per Pagaments.